src/auth/s2s/s2s.class.spec.ts

```diff
@@ -0,0 +1,46 @@
+// import axios from 'axios'
+// import { AxiosInstance } from 'axios'
+// import { AxiosResponse } from 'axios'
+// import chai from 'chai'
+// import { expect } from 'chai'
+// // eslint-disable-next-line @typescript-eslint/camelcase
+// import jwt_decode from 'jwt-decode'
+// import sinon from 'sinon'
+// import sinonChai from 'sinon-chai'
+// import { mockReq, mockRes } from 'sinon-express-mock'
+import { S2SConfig } from './s2sConfig.interface'
+import { S2SAuth } from './s2s.class'
+// import * as serviceAuth from './serviceAuth'
+
+// chai.use(sinonChai)
+
+describe('S2SAuth', () => {
+    let s2sAuth: S2SAuth
+    const s2sConfig = {
+        microservice: 'rpx-xui',
+        s2sEndpointUrl: 'http://s2s.local',
+        s2sSecret: 'topSecret',
+    } as S2SConfig
+    // const token = 'abc123'
+
+    beforeEach(() => {
+        s2sAuth = new S2SAuth()
+        s2sAuth.configure(s2sConfig)
+        // sinon.stub(serviceAuth, 'postS2SLease').resolves(token)
```

Comment:
know we know how to properly mock any API calls, can we please re-enable the tests and implement?
E.g. here
```
jest.spyOn(serviceAuth, 'postS2SLease').mockImplementation(() => Promise.resolve(token))
```

Reply:
Sure - will work on this as part of a new PR for the ticket covering remaining unit test activities.